Foundation_statute.md

```diff
@@ -0,0 +1,146 @@
+4. Fundacja może tworzyć fundusze, w tym o charakterze celowym, zgodnie z wolą ofiarodawcy, sygnowane jego nazwiskiem lub nazwą.
+5. Przyjęcie przez Fundację darowizny lub spadku nie może spowodować przejęcia przez Fundację długów przewyższających wartość przyjmowanego mienia.
+6. W sprawach przyjęcia darowizn i dziedziczenia oświadczenia wymagane przepisami prawa składa Zarząd Fundacji.
+
+§12 Działalność gospodarcza
+1. Fundacja prowadzi działalność gospodarczą.
+2. Przychód z odpłatnej działalności statutowej służy wyłącznie realizacji celów Fundacji.
+
+§13 Księgi rachunkowe
+Fundacja prowadzi gospodarkę finansową i rachunkowość według zasad obowiązujących osoby prawne.
+
+§14 Powiernictwo środków
+Fundacja może gromadzić swoje fundusze w walucie polskiej oraz w walutach obcych we właściwych bankach zgodnie z przepisami prawa.
+
+§15 Odpowiedzialność za zobowiązania
+Fundacja odpowiada za swoje zobowiązania całym majątkiem.
+
+Rozdział IV: Organy Fundacji
+
+§16 Zarząd
+1. Organem Fundacji jest Zarząd Fundacji.
+2. Zarząd kieruje działaniami Fundacji i reprezentuje ją na zewnątrz.
+
+§17 Fundatorzy
+1. Fundator do wykonywania swoich uprawnień może powołać pełnomocnika.
+2. W przypadku śmierci Fundatora uprawnienia przewidziane w niniejszym statucie dla Fundatora, wykonuje jego pełnoletnie dziecko, łącznie pełnoletnie dzieci lub do czasu uzyskania pełnoletności ich prawni opiekunowie. Jeżeli Fundator nie posiadał dzieci, jego obowiązki wykonuje żyjący małżonek, albo łącznie rodzeństwo Fundatora, albo łącznie pełnoletnie dzieci rodzeństwa Fundatora lub do czasu uzyskania pełnoletności ich prawni opiekunowie.
+
+§18 Członkowie Zarządu 
+1. Członkowie pierwszego Zarządu Fundacji są powołanie przez Fundatorów w drodze zgodnego oświadczenia w akcie powołania.
+2. Fundator może zostać członkiem Zarządu Fundacji.
+3. Zarząd Fundacji liczy od dwóch do pięciu członków.
+4. Członkowie Zarządu Fundacji mogą otrzymywać wynagrodzenie z tytułu pełnienia swoich funkcji.
+5. Członkostwo w Zarządzie Fundacji ustaje w przypadku odwołania, pisemnej rezygnacji z członkostwa lub śmierci członka Zarządu.
+
+§19 Zadania Zarządu
+1. Zarząd Fundacji kieruje działalnością Fundacji i reprezentuje ją na zewnątrz.
+2. Do zadań Zarządu należy w szczególności:
+	a) kierowanie bieżącą działalnością Fundacji;
+	b) realizacja celów statutowych Fundacji,
+	c) uchwalanie rocznych i wieloletnich planów działania Fundacji,
+	d) przygotowywanie i uchwalanie sprawozdań z działalności Fundacji, w tym sprawozdań finansowych,
+	e) zarządzanie majątkiem Fundacji,
+	f) powoływanie i odwoływanie członków zarzadu.
+3. Zarząd Fundacji podejmuje decyzje na posiedzeniach w formie uchwał zwykłą większością głosów.
+4. O posiedzeniu muszą być powiadomieni wszyscy członkowie Zarządu poprzez wiadomość e-mail.
```

Comment:
Wydaje mi się, że to dosyć ogólne stwierdzenie. Na jaki e-mail zawiadomienie powinno być skierowane? Kiedy e-mail był podany? Czy jest gdzieś zapisany? Kto ma obowiązek go aktualizować? Czy wysłanie zawiadomienia na nieaktualny e-mail jest skuteczne? Z jakim wyprzedzeniem należy zawiadomić członków zarządu o posiedzeniu? Czy zawiadomienie musi być opatrzone podpisem elektronicznym? Czy odbiorca ma obowiązek potwierdzić odbiór? W jakim terminie musi potwierdzić? Czy niepotwierdzenie odbioru zawiadomienia w terminie umożliwiającym zapoznanie się z jego treścią, można uznać na jakimś etapie za skuteczne?